squad/api/rest.py

```diff
@@ -945,6 +964,12 @@ def metrics(self, request, pk=None):
         serializer = MetricSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+    @action(detail=True, methods=['get'], suffix='status')
+    def status(self, request, pk=None):
+        status = self.get_object().status.all()
```

Comment:
Here is better to display only the status which, `suite=None`, I pulled your PR and it's showing all statuses by default.

![Screenshot from 2020-03-18 09-58-22](https://user-images.githubusercontent.com/2254825/77002057-ac1f3480-6939-11ea-956d-052aec4cacec.png)

Reply:
the reason i changed it to show all suites is what @mwasilew said last Monday when he said it would be better to query/filter by suite_id in some cases! Milosz?

Reply:
The filter is wrong. By default you should be using get(suite=None). 'suite' should be an additional parameter of this endpoint and only if suite is present it should be used to filter the proper data. This is exactly what I said.

Reply:
@chaws @mwasilew reset and repushed.

Reply:
how do I pass additional parameters to status() ?